### ISSUE TITLE
When will split2 and split3 of the diro dataset be released

### ISSUE THREAD

**Issue by qjh666888** [YanxingLiu/SAE-FSDet#14]
When will split2 and split3 of the diro dataset be released?

**Comment by YanxingLiu** [comment]
You can simply modify the split num in the config of split1.

**Comment by qjh666888** [comment]
![image](https://github.com/user-attachments/assets/60ee5885-3b0d-477a-9318-d74274ce684c)
![image](https://github.com/user-attachments/assets/d3c56171-1d11-48fa-9dbf-094496df2530)
I followed your method and modified spilt2, which can run normally. However, when I modified split3, there was a problem with nan during training. What should I do？

**Comment by YanxingLiu** [comment]
> ![image](https://private-user-images.githubusercontent.com/73637597/399394889-60ee5885-3b0d-477a-9318-d74274ce684c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU2MTA4MTcsIm5iZiI6MTczNTYxMDUxNywicGF0aCI6Ii83MzYzNzU5Ny8zOTkzOTQ4ODktNjBlZTU4ODUtM2IwZC00NzdhLTkzMTgtZDc0Mjc0Y2U2ODRjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjMxVDAyMDE1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlhNTg1M2I3YmVmMzE1NTQ5MTE1MDc0ZDBiOWU5Y2E5Zjk1N2IyMWZhZjg3MzZkODM3ZDgxNjNmM2M5ZDZmY2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.jxqBL4wvMIbb9LlJjB2EVQiVuMBGP_UKFQyg-jjMqX8) ![image](https://private-user-images.githubusercontent.com/73637597/399394901-d3c56171-1d11-48fa-9dbf-094496df2530.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU2MTA4MTcsIm5iZiI6MTczNTYxMDUxNywicGF0aCI6Ii83MzYzNzU5Ny8zOTkzOTQ5MDEtZDNjNTYxNzEtMWQxMS00OGZhLTlkYmYtMDk0NDk2ZGYyNTMwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjMxVDAyMDE1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY5YjNjNDZlODQ3NGNhMjVhNjlmM2ExMTA5ZjZhZGQ4MGM3NTA3NzRhNDdkZDg4ODM5YWNlMGI3OTEzNGVjY2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.wcV7hhE-nugGoplkekNWaY7XtJr_YA7Alp8OWSpU64U) I followed your method and modified spilt2, which can run normally. However, when I modified split3, there was a problem with nan during training. What should I do？

During my training process, I found that there are very few boxes with a length or width of 0 in the DIOR dataset. If these parts are not removed during training, it will cause nan. It can be removed by min_box_area.

**Comment by YanxingLiu** [comment]
> ![image](https://private-user-images.githubusercontent.com/73637597/399394889-60ee5885-3b0d-477a-9318-d74274ce684c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU2MTA4MTcsIm5iZiI6MTczNTYxMDUxNywicGF0aCI6Ii83MzYzNzU5Ny8zOTkzOTQ4ODktNjBlZTU4ODUtM2IwZC00NzdhLTkzMTgtZDc0Mjc0Y2U2ODRjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjMxVDAyMDE1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlhNTg1M2I3YmVmMzE1NTQ5MTE1MDc0ZDBiOWU5Y2E5Zjk1N2IyMWZhZjg3MzZkODM3ZDgxNjNmM2M5ZDZmY2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.jxqBL4wvMIbb9LlJjB2EVQiVuMBGP_UKFQyg-jjMqX8) ![image](https://private-user-images.githubusercontent.com/73637597/399394901-d3c56171-1d11-48fa-9dbf-094496df2530.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU2MTA4MTcsIm5iZiI6MTczNTYxMDUxNywicGF0aCI6Ii83MzYzNzU5Ny8zOTkzOTQ5MDEtZDNjNTYxNzEtMWQxMS00OGZhLTlkYmYtMDk0NDk2ZGYyNTMwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjMxVDAyMDE1N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY5YjNjNDZlODQ3NGNhMjVhNjlmM2ExMTA5ZjZhZGQ4MGM3NTA3NzRhNDdkZDg4ODM5YWNlMGI3OTEzNGVjY2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.wcV7hhE-nugGoplkekNWaY7XtJr_YA7Alp8OWSpU64U) I followed your method and modified spilt2, which can run normally. However, when I modified split3, there was a problem with nan during training. What should I do？

In addition, unreasonable learning rates may also cause this phenomenon. The learning rate in my config is for single-card configuration. If you use multiple cards, you need to modify the learning rate and corresponding hyperparameters according to the linear scaling principle.

**Comment by qjh666888** [comment]
I couldn't find the code for min_ox_rea. Where is it? I am using a single GPU, and only split3 is experiencing nan issues, while the other splits are not.

**Comment by qjh666888** [comment]
Excuse me, is it enough to just change split1 to split2? Do you need to modify the learning rate

**Comment by qjh666888** [comment]
Excuse me, when I am fine-tuning, do I just need to modify split1 to split2? Do you need to modify the learning rate

**Comment by qjh666888** [comment]
In the fine-tuning of split2, I am unable to achieve the accuracy of the paper. May I ask what parameters I need to modify？
